dev/ci/nix/coq.nix

```diff
@@ -5,5 +5,4 @@ let coq = callPackage wd { buildDoc = false; doInstallCheck = false; coq-version
 coq.overrideAttrs (o: {
   name = "coq-local-${branch}";
   src = fetchGit "${wd}";
-  enableParallelBuilding = true;
```

Comment:
Didn't you forget to add this to `default.nix`?

Reply:
No. This affects the arguments of `make`. No parallelism between the various `Makefile.dune` targets is expected. `dune` has its own heuristics.